docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+
+= Resolving issues with the {ProductName} extension
+
+You can resolve issues by using a Quick Fix or by editing the code yourself.
+
+.Procedure
+
+. Resolve issues detected by the {ProductName} extension by performing one of the following actions:
+
+* If the issue displays the Quick Fix icon (image:vs_optional.png[Optional or Quick Fix]), you can right-click the issue and select *Apply All QuickFixes*.
```

Comment:
Are you sure that is what the context menu says? I recall "Preview Quick Fix" and "Apply Quick Fix". Check this.

Reply:
Yes "Preview Quick Fix" and "Apply Quick Fix" is correct.